feature/86cw0vg9n-artist-gallery-post-database-setup

viggy21 opened the pull request:
Purpose: Initial set up of Post collection in the database
Involved:

- Creating the PostCollection
- Set up of methods for posts: adding posts, fetching posts based on post ID, updating posts, deleting posts based on post ID
- Post publication for posts with the artistUsername matching the provided username

nethbtju commented on behind-the-veil-siteroot/imports/ui/App.jsx:
I think this might need a rebase some of work is out of sync

nethbtju commented on behind-the-veil-siteroot/imports/api/methods/posts.js:
Some test cases for these would be good since its not being called anywhere yet

nethbtju commented:
I think you might need to rebranch and move your changes to a new branch - this one is reverting to an old state of develop

viggy21 commented:
Hi Neth, I've created a new branch based on develop (https://github.com/Monash-FIT3170/Behind-The-Veil/tree/feature/86cw0vg9n-artist-gallery-post-database-setup-v2) to replace this one. I added test cases for the post methods and also fixed one of the test cases for booking (test case for adding the booking wasn't working)
